content/applications/general/developer_mode.rst

```diff
@@ -11,6 +11,9 @@ Activate through the Settings
 
 Go to :menuselection:`Settings --> Activate the developer mode`.
 
+.. note::
+   You need to have at least one app installed for this menu option to appear.
```

Comment:
Hi! Can we change this line to: 

`Note: At least one application needs to be installed for the developer menu option to appear.`

Let's remove the familiar second person language and make it less personal.

Reply:
@javl Are you okay with the above changes? If so, can you update the RST and we will get the doc updated :) Thanks, Tim